elixir/apps/web/lib/web/live/settings/dns.ex

```diff
@@ -32,24 +32,28 @@ defmodule Web.Settings.DNS do
       <:title>
         DNS
       </:title>
+
+      <:action>
+        <.docs_action path="/deploy/dns" />
+      </:action>
+
       <:help>
-        Configure the default resolver used by connected Clients.
-        Queries for Resources will <strong>always</strong>
-        use Firezone's internal DNS.
-        All other queries will use the DNS servers configured here or the Client's
-        system resolvers if no servers are configured.
-        <p class="mt-2">
-          <.website_link path="/kb/deploy/dns">
-            Read more about configuring DNS in Firezone.
-          </.website_link>
-        </p>
+        Configure the upstream resolver used by connected Clients.
+        Queries for Resources will <strong>always</strong> use Firezone's internal DNS.
+        All other queries will use the resolvers configured here or the Client's
+        system resolvers if none are configured.
       </:help>
       <:content>
         <div class="max-w-2xl px-4 py-8 mx-auto">
           <.flash kind={:success} flash={@flash} phx-click="lv:clear-flash" />
           <h2 class="mb-4 text-xl text-neutral-900">Client DNS</h2>
-          <p class="mb-4 text-neutral-500">
-            DNS servers will be used in the order they are listed below.
+          <p :if={Enum.any?(@form.source.changes)} class="mb-4 text-neutral-500">
+            Upstream resolvers will be used by Clients in the order they are listed below.
+          </p>
+
+          <p :if={Enum.empty?(@form.source.changes)} class="text-neutral-500">
+            No upstream resolvers have been configured. Click <strong>New Resolver</strong>
+            to add one.
```

Comment:
I don't think this will work as expected.  If a DNS server has already been configured in a prior session this message will still be displayed, because there won't be any changes, but it will display the existing DNS server.

### Example:
<img width="675" alt="Screenshot 2024-10-13 at 11 01 26 PM" src="https://github.com/user-attachments/assets/10ebb19d-7460-460e-b411-c7c13f7044fe">

Reply:
Good catch! I wrote that pretty late 😅 should I push a quick fix?

Reply:
I already pushed a fix, no worries :)